driver/kubernetes/factory.go

```diff
@@ -108,7 +108,9 @@ func (f *factory) New(ctx context.Context, cfg driver.InitConfig) (driver.Driver
 			if err != nil {
 				return nil, err
 			}
-			deploymentOpt.Image = bkimage.DefaultRootlessImage
+			if deploymentOpt.Image == bkimage.DefaultImage {
```

Comment:
This doesn't look very safe. Instead do the `_, isImage := cfg.DriverOpts["image"]; isImage` check

Reply:
Ah, yep that does seem better.